src/Operators/remapping.jl

```diff
@@ -290,3 +290,103 @@ function local_weights(space::AbstractSpace)
     wj = space.local_geometry.WJ
     return vec(parent(wj))
 end
+
+## RegularLatLong interpolation
+struct RLLRemap{
+    T <: Meshes.RegularLatLongMesh,
+    S <: AbstractSpace,
+    M <: AbstractMatrix,
+    E <: AbstractVector,
+    P <: AbstractVector,
+}
+    target::T # rll mesh
+    source::S # source space
+    map::M # linear mapping operator
+    elems::E # elements associated with target pts
+    p::P # coordinate permutation vector
+end
+
+function RLLRemap(target_mesh::Meshes.RegularLatLongMesh, source_space)
+
+    source_mesh = Spaces.topology(source_space).mesh
+    latitude, longitude = target_mesh.lat, target_mesh.long
+
+    # store latlong coords and associated elements
+    coords = []
+    for lat in latitude
+        for long in longitude
+            coord = Geometry.LatLongPoint(lat, long)
+            elem = Meshes.containing_element(source_mesh, coord)
+            push!(coords, (coord = coord, elem = elem))
+        end
+    end
+
+    # sort coords by element
+    p = sortperm(coords, by = coord -> coord.elem)
+    permute!(coords, p)
+    coords = map(coords, 1:length(coords)) do coord, i
+        (coord..., idx = i)
+    end
+    # get unique elements and the indices of the first associated coords
+    elems = map(x -> (elem = x.elem, idx = x.idx), unique(x -> x.elem, coords))
+
+    QS_s = Spaces.quadrature_style(source_space)
+    Nq_s = Quadratures.degrees_of_freedom(QS_s)
+    # rows - for each rll coord, cols - for each node in corresponding elem
+    op = spzeros(length(coords), Nq_s^2)
```

Comment:
should be a dense matrix, also switch order of indices (as they will be memory aligned)

Reply:
Actually, may be better to do it as `zeros(Nq_s, Nq_s, length(coords))`